src/Disks/tests/gtest_disk_hdfs.cpp

```diff
@@ -0,0 +1,156 @@
+#include <gtest/gtest.h>
```

Comment:
Looks like this test can be run only on your local machine. Am I right?

Reply:
Yes, because it is questionable now whether it is possible to run gtest with hdfs server in CI. Not sure what can be done with that..

Reply:
What exactly integration test can't check that can gtest?
As a baseline you can use merge_tree_s3 integration test. It covers all manipulations with DiskS3.

Reply:
ok, I added multiple integration tests for merge tree over hdfs and some other. Also I fixed all comments, please take a look.